string source
string author
string title
string description
string url
string url_to_image
string published_at
string content
